src/frontend/screens/MapScreen/CurrentTrack/CurrrentTrackMapLayer.tsx

```diff
@@ -1,30 +1,18 @@
 import {LineJoin, LineLayer, ShapeSource} from '@rnmapbox/maps';
 import {usePersistedTrack} from '../../../hooks/persistedState/usePersistedTrack';
 import * as React from 'react';
-import {useLocation} from '../../../hooks/useLocation';
 import {convertToLineString} from '../../../lib/utils';
 
 export const CurrentTrackMapLayer = () => {
   const locationHistory = usePersistedTrack(state => state.locationHistory);
   const isTracking = usePersistedTrack(state => state.isTracking);
-  const {location} = useLocation({maxDistanceInterval: 3});
-  const finalLocationHistory = location?.coords
-    ? [
-        ...locationHistory,
-        {
-          latitude: location.coords.latitude,
-          longitude: location.coords.longitude,
-          timestamp: new Date().getTime(),
-        },
-      ]
-    : locationHistory;
```

Comment:
because tracks was previously not updating often, I think this was done to keep the line up to date with the most current location. This is no longer a problem as the tracks are updating more often.